buildbot/master/files/config/factories.py

```diff
@@ -114,13 +114,16 @@ def make_step(self, command):
                        logEnviron=False),
 ])
 
+def make_cd_string(command):
+    return "cd /c/buildbot/slave/windows/build ; " + command
+
 windows = ServoFactory([
     # TODO: convert this to use DynamicServoFactory
     # We need to run each command in a bash login shell, which breaks the
     # heuristics used by DynamicServoFactory.make_step
-    steps.Compile(command=["bash", "-l", "-c", "./mach build -d -v"],
+    steps.Compile(command=["bash", "-l", "-c", ""], make_cd_string("./mach build -d -v"),
```

Comment:
Actually, this could have been caught via `./test.py sls.buildbot.master` inside Vagrant - there's some extra setup that's required but I've been using it and it's pretty helpful. I'm trying to add local tests where I can :)